src/main/java/evm/vmbase.java

```diff
@@ -0,0 +1,80 @@
+package evm;
+
+import org.ethereum.vm.DataWord;
+//import main.java.evm.BlockStoreMock;
+import org.ethereum.vm.client.BlockStore;
+import org.ethereum.vm.client.Repository;
+//import main.java.evm.RepositoryMock;
+
+//import org.ethereum.vm.program.Program;
+import org.ethereum.vm.program.invoke.ProgramInvokeImpl;
+//import org.ethereum.vm.util.HexUtil;
+//import org.junit.Before;
```

Comment:
we should not merge commented code with the master branch anymore, please check for it entirely.